### PR TITLE
Fixed post Author Badge on Comment template misplacing on Mobile/Tablet

### PR DESCRIPTION
**Pull request by mmaumio**

### CODE REVIEW

File: sass/site/primary/_comments.scss [hunk 1]
```diff
@@ -217,7 +217,10 @@
 			height: 18px;
 			position: absolute;
 			background: lighten( $color__link, 8% );
-			right: calc(100% + #{$size__spacing-unit * .25});
+			right: calc(100% - #{$size__spacing-unit * 3});
+			@include media(tablet) {
```
Comment (allancole) [inline sass/site/primary/_comments.scss]: This looks good but could you move the media query mixin chunk to its own line below this group of rules? It should probably start at line `227`.

See how the media query mixin appears in other places for examples. Thanks!

Comment (mmaumio) [reply sass/site/primary/_comments.scss]: Yeah. Done.
Please check it's right.

**Comment by kjellr** [comment]
Thanks, @mmaumio! I made just one minor adjustment, and this should be good to go.

Before:
screen shot 2018-10-24 at 11 54 20 am


After:
screen shot 2018-10-24 at 11 54 35 am

**Comment by mmaumio** [comment]
@kjellr That looks great.
Thanks

**Comment by allancole** [comment]
Looks good! Thanks @kjellr & @mmaumio

**Pull request merged**